Register Subdomain: lknows

lknows opened the pull request:
Added the domain: lknows

lknows committed:
feat: Added domain programatically

lknows committed:
Update lknows.json

iop3 reviewed (changes requested):
wheres the website? and this isn't the correct json

lknows committed:
Update lknows.json

Correct accidental url, added description, repo.
Hopefully this is correct. sorry I'm new and slow thank you for helping me

lknows commented:
> wheres the website? and this isn't the correct json

Is this json correct now or am I editing wrong entirely wrong directory?

hum-an commented:
this is good just wait for other people to approve and merge :)

MaskDuck committed:
Update domains/lknows.json
